### DISCUSSION TITLE
Welcome to code-narrator Discussions!

### DISCUSSION THREAD

**Discussion by ingig** [ingig/code-narrator#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Deepak0949** [comment]
Will this repo convert my existing local repo code into one document with explained code using AI?

**Reply by ingig** [reply]
Something like that, it creates a readme file for you project, then traverses through each js file and creates a doc for each. Each folder gets its own readme

**Reply by Deepak0949** [reply]
so, it will go through all my folder and if .py file will also be in folder . it will create readme file with code explanation? Correct me if i'm wrong then. I'm getting error while I'm executing npx code-narrator. Error: Cannot find module './src/documentation/plugins/builders/Configuration/ConfigurationBuilder'
Require stack:

**Reply by ingig** [reply]
Check out this: https://github.com/ingig/code-narrator/issues/10

Using sudo seems to fix it: sudo npm i code-narrator -D

**Reply by Deepak0949** [reply]
It's working if I'm using command ' npm run start' but if i'm giving other repo path not generating document for other repo. is there any other way to point other local repo. please guide me.  My command for other repo npm run start -- --path D:\\auxiliary-repo

**Reply by ingig** [reply]
--path is not a supported parameter. If I remember correctly you need to install the npm package in your project. I am a bit rusty, I haven't touched this project for long time.

**Reply by Deepak0949** [reply]
Just tell me how I can use your code in my repo?

**Reply by Deepak0949** [reply]
As I mentioned earlier if I will put some folder with code in your repo. It's working.

**Reply by ingig** [reply]
As I don't have the project setup I cannot guarantee this, but you I believe you should install it the where you have your code repo

Navigate to where your code repo is, then
```
npm i code-narrator
```

then it is installed in your project, then you should be able to run it from the path of your project

Again want to emphases, I haven't touch on this project for long time, so I am not sure. Just going by memory now
